src/improvements/tasks/sep/006-unpause-superchainConfig/README.md

```diff
@@ -0,0 +1,35 @@
+# 007-Unpause the SuperchainConfig
+
+Status: [READYTOSIGN]()
+
+## Objective
+This would be part of the unpausing of sepolia following the pause of the superchain since we are testing that U13 upgrade for the Pause mechanism is working as expected.
+This task **unpause** the SuperchainConfig contract. This will first check if the Superchain is `paused`. If this is not `paused`we will skip the execution of this task.
```

Comment:
```suggestion
This task **unpause** the SuperchainConfig contract. This will first check if the Superchain is `paused`. If this is not `paused` we will skip the execution of this task.
```